<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>221</width>
      <height>561</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>44</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>116</red>
          <green>170</green>
          <blue>130</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>44</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>116</red>
          <green>170</green>
          <blue>130</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>116</red>
          <green>170</green>
          <blue>130</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>116</red>
          <green>170</green>
          <blue>130</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>219</width>
       <height>559</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>53</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>53</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Century</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>İŞLEM SEÇİNİZ</string>
      </property>
      <property name="frame">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonAccuracy">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>440</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Accuracy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonLoss">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Loss</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxSinif">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>Sınıflandırıcı seç</string>
      </property>
      <item>
       <property name="text">
        <string>Sınıflandırıcı seç</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>K-Neighbours</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Logistic Regression</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SVM</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxModelSec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>Model Seç</string>
      </property>
      <item>
       <property name="text">
        <string>Model Seç</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ResnetB4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EfficientNetB5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>VGG16</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Our Model</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBoxKFold">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentText">
       <string>K-Fold seçiniz</string>
      </property>
      <item>
       <property name="text">
        <string>K-Fold seçiniz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>10</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="buttonSinifCalistir">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Çalıştır</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonModelEgit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Model Eğit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonKFoldUygula">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Uygula</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonVeriCokla">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Veri Çoklama</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbEarlyStopping">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Early Stopping</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbModelCheckpoint">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Model Checkpoint</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtEpochSayisi">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Epoch Sayısı Girin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtBatchSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nirmala UI</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Batch Size Sayısı Girin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTespit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Resim Tespiti</string>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonCM">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Confusion Matrix</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Veriseti_Yukle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Veriseti Yükle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResimSec">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Resim seç</string>
    </property>
   </widget>
   <widget class="QPushButton" name="h5Sec">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>h5 dosyası sec</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="imageBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>251</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
